docs/requirement-sets/outlook/outlook-api-requirement-sets.md

```diff
@@ -116,7 +116,7 @@ Add-ins are supported in Outlook on the following platforms.
 || Microsoft 365 subscription | [1.1](requirement-set-1.1/outlook-requirement-set-1.1.md), [1.2](requirement-set-1.2/outlook-requirement-set-1.2.md), [1.3](requirement-set-1.3/outlook-requirement-set-1.3.md), [1.4](requirement-set-1.4/outlook-requirement-set-1.4.md), [1.5](requirement-set-1.5/outlook-requirement-set-1.5.md), [1.6](requirement-set-1.6/outlook-requirement-set-1.6.md), [1.7](requirement-set-1.7/outlook-requirement-set-1.7.md), [1.8](requirement-set-1.8/outlook-requirement-set-1.8.md)<sup>4</sup>, [1.9](requirement-set-1.9/outlook-requirement-set-1.9.md)<sup>4</sup>, [1.10](requirement-set-1.10/outlook-requirement-set-1.10.md)<sup>4</sup>, [1.11](requirement-set-1.11/outlook-requirement-set-1.11.md)<sup>4</sup>, [1.12](requirement-set-1.12/outlook-requirement-set-1.12.md)<sup>4</sup>, [1.13](requirement-set-1.13/outlook-requirement-set-1.13.md)<sup>4</sup>, [1.14](requirement-set-1.14/outlook-requirement-set-1.14.md)<sup>4</sup><br>[DialogAPI 1.1](../common/dialog-api-requirement-sets.md)<br>[DialogAPI 1.2](../common/dialog-api-requirement-sets.md)<br>[DialogOrigin 1.1](../common/dialog-origin-requirement-sets.md)<br>[IdentityAPI 1.3](../common/identity-api-requirement-sets.md)<sup>3</sup><br>[NestedAppAuth 1.1](../common/nested-app-auth-requirement-sets.md)<br>[OpenBrowserWindowApi 1.1](../common/open-browser-window-api-requirement-sets.md) |
 || retail perpetual Outlook 2016 and later | [1.1](requirement-set-1.1/outlook-requirement-set-1.1.md), [1.2](requirement-set-1.2/outlook-requirement-set-1.2.md), [1.3](requirement-set-1.3/outlook-requirement-set-1.3.md), [1.4](requirement-set-1.4/outlook-requirement-set-1.4.md), [1.5](requirement-set-1.5/outlook-requirement-set-1.5.md), [1.6](requirement-set-1.6/outlook-requirement-set-1.6.md), [1.7](requirement-set-1.7/outlook-requirement-set-1.7.md), [1.8](requirement-set-1.8/outlook-requirement-set-1.8.md)<sup>4</sup>, [1.9](requirement-set-1.9/outlook-requirement-set-1.9.md)<sup>4</sup>, [1.10](requirement-set-1.10/outlook-requirement-set-1.10.md)<sup>4</sup>, [1.11](requirement-set-1.11/outlook-requirement-set-1.11.md)<sup>4</sup>, [1.12](requirement-set-1.12/outlook-requirement-set-1.12.md)<sup>4</sup>, [1.13](requirement-set-1.13/outlook-requirement-set-1.13.md)<sup>4</sup>, [1.14](requirement-set-1.14/outlook-requirement-set-1.14.md)<sup>4</sup><br>[DialogAPI 1.1](../common/dialog-api-requirement-sets.md)<br>[DialogAPI 1.2](../common/dialog-api-requirement-sets.md)<br>[DialogOrigin 1.1](../common/dialog-origin-requirement-sets.md)<br>[IdentityAPI 1.3](../common/identity-api-requirement-sets.md)<sup>3</sup><br>[OpenBrowserWindowApi 1.1](../common/open-browser-window-api-requirement-sets.md) |
```

Comment:
I see nested app auth is available in VL perpetual Outlook 2024. It still isn't available in retail perpetual? Looks like it's added in the "Office on Windows - perpetual" column in the nested app auth reqt. sets page...